crates/vault-registry/src/lib.rs

```diff
@@ -861,22 +921,36 @@ impl<T: Config> Module<T> {
     }
 
     /// Automatically liquidates all vaults under the secure threshold
-    fn liquidate_undercollateralized_vaults() -> u32 {
+    pub fn liquidate_undercollateralized_vaults(
+        liquidation_target: LiquidationTarget,
```

Comment:
this parameter name is misleading - from the naming I expected it to only liquidate vaults that matched the target, which is not the case

Reply:
It's actually the implementation that's wrong, thanks for pointing it out.